.ci_support/migrations/python310.yaml

```diff
@@ -0,0 +1,34 @@
+migrator_ts: 1634137107
+ __migrator:
```

Comment:
Everything from here down is indented by one too many spaces.

Reply:
Wooops... Copy/paste from GitHub web 🙈 Will fix it...